ui/shared/components/panel/FamilyReads.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Segment, Icon } from 'semantic-ui-react'
+
+import { getIndividualsByGuid, getIGVSamplesByFamily, getFamiliesByGuid } from 'redux/selectors'
+import PedigreeIcon from '../icons/PedigreeIcon'
+import IGV from '../graph/IGV'
+import { ButtonLink } from '../StyledComponents'
+import { VerticalSpacer } from '../Spacers'
+import { getLocus } from './variants/Annotations'
+
+const CRAM_PROXY_TRACK_OPTIONS = {
```

Comment:
The top of this file is directly copy-pasted form the old FamlyVariantReads component, so I didn't change anything and it doesn't need review